int32 ID
int32 company
int32 device
int32 softwareVersion
int32 bus
